.gitlab/pipeline/check.yml

```diff
@@ -113,12 +113,41 @@ test-rust-feature-propagation:
   script:
     - |
       export RUST_LOG=remote-ext=debug,runtime=debug
-      echo "---------- Running try-runtime for ${NETWORK} ----------"
-      time cargo install --locked --git https://github.com/paritytech/try-runtime-cli --rev a93c9b5abe5d31a4cf1936204f7e5c489184b521
-      time cargo build --release --locked -p "$NETWORK"-runtime --features try-runtime
+      echo "---------- Installing try-runtime-cli ----------"
+      time cargo install --locked --git https://github.com/paritytech/try-runtime-cli --tag v0.3.0
+      echo "---------- Building ${PACKAGE} runtime ----------"
+      time cargo build --release --locked -p "$PACKAGE" --features try-runtime
+      echo "---------- Executing `on-runtime-upgrade` for ${NETWORK} ----------"
       time try-runtime \
-          --runtime ./target/release/wbuild/"$NETWORK"-runtime/target/wasm32-unknown-unknown/release/"$NETWORK"_runtime.wasm \
-          on-runtime-upgrade --checks=pre-and-post live --uri wss://${NETWORK}-try-runtime-node.parity-chains.parity.io:443
+          --runtime ./target/release/wbuild/"$PACKAGE"/target/wasm32-unknown-unknown/release/"$WASM" \
+          on-runtime-upgrade --checks=pre-and-post ${EXTRA_ARGS} live --uri ${URI}
+
+# Check runtime migrations for relay chains
+check-runtime-migration-polkadot:
+  stage: check
+  extends:
+    - .docker-env
+    - .test-pr-refs
+    - .check-runtime-migration
+  variables:
+    NETWORK: "polkadot"
+    PACKAGE: "polkadot-runtime"
+    WASM: "polkadot_runtime.wasm"
+    URI: "wss://polkadot-try-runtime-node.parity-chains.parity.io:443"
+    EXTRA_ARGS: "--no-weight-warnings"
+
+check-runtime-migration-kusama:
+  stage: check
+  extends:
+    - .docker-env
+    - .test-pr-refs
+    - .check-runtime-migration
+  variables:
+    NETWORK: "kusama"
+    PACKAGE: "staging-kusama-runtime"
+    WASM: "staging_kusama_runtime.wasm"
+    URI: "wss://kusama-try-runtime-node.parity-chains.parity.io:443"
+    EXTRA_ARGS: "--no-weight-warnings"
```

Comment:
These 2 jobs were removed in [this PR](https://github.com/paritytech/polkadot-sdk/pull/1316). @ggwpez Should we return them back?

Reply:
I think we have to should these to the runtimes repo.